src/models/Act.ts

```diff
@@ -1,6 +1,6 @@
 export const Act: Object = {
   name: String,
-  localizedNames: String,
+  localizedNames: String, // Field is excluded from the response when a locale is set
```

Comment:
Adding comments like so usually doesn't require it's own commit, especially when it's follows immediately after the commit adding functionality.

`git commit --amend` allows you to add to the previous commit. Do everything else the same, make changes, git add, and replace your commit with the amend command.
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History